mgmt/ProcessManager.cc

```diff
@@ -387,6 +387,12 @@ ProcessManager::handleMgmtMsgFromLM(MgmtMessageHdr *mh)
   case MGMT_EVENT_CLEAR_STATS:
     signalMgmtEntity(MGMT_EVENT_CLEAR_STATS);
     break;
+  case MGMT_EVENT_HOST_STATUS_UP:
+    signalMgmtEntity(MGMT_EVENT_HOST_STATUS_UP, data_raw, mh->data_len);
```

Comment:
#3214 removes `signalMgmtEntity` logic. You can directly send the event rather than enqueuing it using `executeMgmtCallback` with the same parameters. So, for example, `executeMgmtCallback(MGMT_EVENT_HOST_STATUS_UP, data_raw, mh->data_len)`